File: Data Structures & Algorithms/duplicate-integer/submission-0.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        var duplicateDict = new Dictionary<int, int>();

        foreach(var num in nums) {
            if(duplicateDict.ContainsKey(num)) {
                return true;
            }

            duplicateDict[num] = num;
        }

        return false;
    }
}

File: Data Structures & Algorithms/sort-an-array/submission-0.py
class Solution:
    def sortArray(self, nums: List[int]) -> List[int]:
        counter = [0] * 10**6

        for num in nums:
            counter[num + 50000] += 1

        result = []

        for i in range(len(counter)):
            for _ in range(counter[i]):
                result.append(i - 50000)

        return result